static/js/components/BokehModels.js

```diff
@@ -3,19 +3,18 @@
 import { input, label, div, span } from "bokehjs/core/dom";
 import { includes } from "bokehjs/core/util/array";
 import * as p from "bokehjs/core/properties";
-import { bk_inline } from "bokehjs/styles/mixins";
-import { bk_input_group } from "bokehjs/styles/widgets/inputs";
+import * as inputs from "bokehjs/styles/widgets/inputs.css";
```

Comment:
```suggestion
import * as bokehInputsCss from "bokehjs/styles/widgets/inputs.css";
```
?

Reply:
This file is a modified version of code in Bokeh (checkbox widget) so I am trying to keep them the same with minimal changes to make it easier to diff.

Reply:
Okay, no problem then.